org.eclipse.lyo.oslc4j.adaptormodel/model/adaptorInterface.ecore

```diff
@@ -178,6 +178,8 @@
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="allowedValue" upperBound="-1"
         eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="isMemberProperty" eType="#//ResourcePropertyIsMemberProperty"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="additionalConstraints"
```

Comment:
This is the main change - introducing an optional property. The rest is generated.

Reply:
It's just a string. No formatting.